app/api/models/ftplinks.py

```diff
@@ -0,0 +1,28 @@
+"""
+    See the NOTICE file distributed with this work for additional information
+    regarding copyright ownership.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+
+from pydantic import BaseModel, Field, HttpUrl, field_serializer
+from typing import Literal
+
+class FTPLink(BaseModel):
+    dataset: Literal['genebuild', 'assembly', 'homologies',
+                     'regulation', 'variation'] = Field(..., validation_alias='datasetType')
+    url: str = Field(..., validation_alias='path')
+    
+    @field_serializer('url')
+    def url_serializer(self, path: str) -> HttpUrl:
+        return f"https://ftp.ebi.ac.uk/pub/databases/ensembl/organisms/{path}"
```

Comment:
Would be good to pass the base URL as an environment variable than hardcoding it.

Reply:
Done